### ISSUE TITLE
Release hash for git-lfs-windows-2.2.1.exe does not match

### ISSUE THREAD

**Issue by larsxschneider** [git-lfs/git-lfs#2454]
The release page for [Git LFS v2.2.1](https://github.com/git-lfs/git-lfs/releases/tag/v2.2.1) presents `3289eb...` as SHA-256 hash for `git-lfs-windows-2.2.1.exe`. If I download the file and calculate the hash then I get `0f70b16...`

**Steps to reproduce:**
```
$ wget https://github.com/git-lfs/git-lfs/releases/download/v2.2.1/git-lfs-windows-2.2.1.exe
$ shasum -p --algorithm 256 git-lfs-windows-2.2.1.exe
```

**Comment by ttaylorr** [comment]
Fixed, thanks!

**Comment by larsxschneider** [comment]
Fixed for 2.2.1 or fixed in general?

2.2.1 still shows the faulty hash: https://github.com/git-lfs/git-lfs/releases/tag/v2.2.1

**Comment by ttaylorr** [comment]
2.2.1 should show the `0f70b16` hash correctly; I'm not sure why it didn't update when I changed it yesterday. Sorry about that!
